config.py

```diff
@@ -65,26 +65,19 @@
 
 # ---------------------------------- MONGO -
 
-
-
 SECRET_KEY 					= os.getenv('SESSION_SECRET', 'Significance')
 
-
 BASIC_AUTH_USERNAME			= os.getenv('BASIC_AUTH_USERNAME', '')
 BASIC_AUTH_PASSWORD			= os.getenv('BASIC_AUTH_PASSWORD', '')
 
 
 TWILIO_ACCOUNT_SID			= os.environ['TWILIO_ACCOUNT_SID']
-TWILIO_AUTH_TOKEN			= os.environ['TWILIO_AUTH_TOKEN']
-TWILIO_NUMBER 				= os.getenv('TWILIO_NUMBER', '+16466473454')
-
-GOOGLE_API_KEY 				= "AIzaSyCJxQK1oDn4U3kbDIK-epf96ckze7fuSHQ"
+TWILIO_AUTH_TOKEN			  = os.environ['TWILIO_AUTH_TOKEN']
+TWILIO_NUMBER 				  = os.environ['TWILIO_NUMBER']
```

Comment:
Why not 
`TWILIO_NUMBER = os.getenv('TWILIO_NUMBER', '+18777130491')`
If you said to use that in the README?